### ISSUE TITLE
Can't install package

### ISSUE THREAD

**Issue by moslemparve** [jlevers/selling-partner-api#9]
- You can only install one version of a package, so only one of these can be installed: vlucas/phpdotenv[dev-master, v4.0.0, ..., 4.2.x-dev, v5.2.0, 5.2.x-dev, v5.3.0].
    - jlevers/selling-partner-api 2.0.0 requires vlucas/phpdotenv ^5.2 -> satisfiable by vlucas/phpdotenv[v5.2.0, 5.2.x-dev, v5.3.0, 5.3.x-dev (alias of dev-master)].

**Comment by jlevers** [comment]
Do you have another version of `vlucas/dotenv` specified in your `composer.json`? If not, could you search your `composer.lock` for `vlucas/dotenv` to check if there's another package you've installed that's requiring a different version of that package?

**Comment by moslemparve** [comment]
No i dont have any version of vlucas/dotenv in my ### composer.json or in ### composer.lock

**Comment by jlevers** [comment]
Do you have it installed globally? You can check with `composer global show`

**Comment by moslemparve** [comment]
No i am not installing globally i am trying to use it in my project which is on Laravel Framework 7.30.4 
can i use it here.?

**Comment by moslemparve** [comment]
![image](https://user-images.githubusercontent.com/40214868/112487013-38aad100-8d9e-11eb-89be-93463f071485.png)

Here is my **composer global show**

**Comment by moslemparve** [comment]
![image](https://user-images.githubusercontent.com/40214868/112487013-38aad100-8d9e-11eb-89be-93463f071485.png)

Here is my **composer global show**

**Comment by jlevers** [comment]
My mistake -- I was referring to the wrong name of the dotenv package. It's actually `vlucas/phpdotenv`. If you search that in your `composer.lock`, I think you'll find that there are several locations where various 4.x versions of that package are required.

I just took a look at the differences between `vlucas/phpdotenv` 4.x and 5.x. Since the differences are minimal, I updated the dependency for this project to allow 4.x versions of that package. Install the latest version of `jlevers/selling-partner-api` (`v2.0.5`) and see if that works for you.
